internal/commands/convert.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/plexsystems/konstraint/internal/rego"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newConvertCommand() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "convert <dir>",
+		Short: "Convert old style annotations to OPA Rich Metadata Annotations",
+		Example: `Convert all found policies inplace
+
+konstraint convert examples`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := "."
+			if len(args) > 0 {
+				path = args[0]
+			}
+
+			return runConvertCommand(path)
+		},
+	}
+
+	return &cmd
+}
+
+func runConvertCommand(path string) error {
+	regos, err := rego.GetAllSeveritiesWithoutImports(path)
+	if err != nil {
+		return fmt.Errorf("get regos: %w", err)
+	}
+
+	for _, r := range regos {
+		logger := log.WithFields(log.Fields{
+			"name": r.Kind(),
+			"src":  r.Path(),
+		})
+
+		if !r.HasLegacyAnnotations() {
+			continue
```

Comment:
Consider adding a logger line here to inform the user the file was skipped and why.